integration/simple_app_test.go

```diff
@@ -416,5 +416,80 @@ func testSimple(t *testing.T, context spec.G, it spec.S) {
 				))
 			})
 		})
+
+		context("BP_DISABLE_SBOM is set to true", func() {
+			var (
+				container1 occam.Container
+				container2 occam.Container
+			)
+
+			it.After(func() {
+				Expect(docker.Container.Remove.Execute(container1.ID)).To(Succeed())
+				Expect(docker.Container.Remove.Execute(container2.ID)).To(Succeed())
+			})
+
+			it("skips SBOM generation", func() {
```

Comment:
Do we also want to check that the Syft, CycloneDX, and SPDX SBOMs are not generated?

Reply:
@ryanmoran  This is a good question. I didn't think we needed it since those files get generated on a different code path. But I can be persuaded to at it :-)
Let me know what you think.

Reply:
Isn't that this code that you modified? https://github.com/paketo-buildpacks/node-engine/pull/472/files#diff-60d42249bb53df5d3ffd44f507aee87a57c2cff5a20f6b1d8403a76d0ce609b0R135-R153.

If we make those changes there, I might suggest that we cover it just as much as we are covering the legacy SBOM format.

Reply:
@ryanmoran I've updated the test. Since the sbom directory doesn't exist, I just checked for [that](https://github.com/paketo-buildpacks/node-engine/pull/472/files#diff-a864a1d24641f3d716c9b563043e4eb4f208754c3172b6527c03d5802d0d30f4R478) instead of checking for all three files. Does this make sense?